src/kirby/components/page/page.component.html

```diff
@@ -5,51 +5,55 @@
         text=""
         [defaultHref]="defaultBackHref"
         icon="assets/kirby/icons/svg/arrow-back.svg"
-        *ngIf="!hideBackButton"
+        [style.visibility]="!hideBackButton ? 'visible' : 'hidden'"
       ></ion-back-button>
     </ion-buttons>
     <ion-title>
-      <div [@visibility]="toolbarTitleVisibility">
+      <div class="hide" [ngClass]="{ 'fade-in': toolbarTitleVisible }">
```

Comment:
Consider simplifying for readability:
```suggestion
      <div class="hide" [class.fade-in]="toolbarTitleVisible">
```